doc/dev/README.md

```diff
@@ -10,19 +11,21 @@
     - [Compile](#compile)
     - [Test](#test)
   - [Build kepler and base multi-arch images](#build-kepler-and-base-multi-arch-images)
-<!--toc:end-->
+  <!--toc:end-->
```

Comment:
needs a newline. otherwise it thinks this comment is part of the indented block above

Reply:
ack